.github/workflows/test.yml

```diff
@@ -1,28 +1,30 @@
-# This is a basic workflow to help you get started with Actions
-
 name: Test
 
-# Controls when the action will run. Triggers the workflow on push or pull request
-# events but only for the master branch
 on:
   push:
     branches: [main]
   pull_request:
     branches: [main]
 
 jobs:
-  build:
-    runs-on: ubuntu-latest
+  test:
+    name: Test / OS ${{ matrix.os }} / Node ${{ matrix.node }} / SDK ${{ matrix.sdk }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        node: ["12"]
+        sdk: [stable, beta, dev]
 
-    container:
-      image: google/dart:latest
+    runs-on: ${{ matrix.os }}
```

Comment:
Hey @phamhieu!

I noticed that `realtime-dart` uses container action to setup dart instead of using [cedx/setup-dart@v2](https://github.com/dart-lang/setup-dart) like all the other Supabase Dart clients. Is there a particular reason for this? Could replacing this container action with official dart action have some effect on something?

Reply:
i don't recall any special reason for this. You are good to go